doc/ExpertOptions.md

```diff
@@ -1470,58 +1482,63 @@ You would write the following manifest:
 {
   "test.input_manifest___": {
```

Comment:
mymanifest.json accepts compact cromwell format.
```suggestion
{
 "test.s": "hello",
 "test.f": "dx://project-aaa:file-xxx"
}
```

Reply:
Are we deprecating the previous manifest format in the favor of cromwell?

Reply:
Suggestion rejected at the meeting 07/21